manifests/w/wahibre/mtn/3.4.2/wahibre.mtn.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: wahibre.mtn
+PackageVersion: 3.4.2
+Installers:
+- InstallerUrl: https://bitbucket.org/wahibre/mtn/downloads/mtn-3.4.2-win64.zip
```

Comment:
Is this the author's official site? It requires an Atlassian login.

Reply:
@stephengillie This is the url to the project, it doesn't require a login https://bitbucket.org/wahibre/mtn/src/master/

The repository is mirrored to gitlab as well where the nicer documentation is. The binaries are published on the bitbucket mirror.
https://gitlab.com/movie_thumbnailer/mtn/-/wikis/home

Reply:
The BitBucket repo claims to be a mirror of the GitLab repo:
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/4f0daa4f-de9e-4b44-b37e-195979b820da)

Reply:
The downloads from the main _gitlab repo_ are pointed to bitbucket. If you look at https://gitlab.com/movie_thumbnailer/mtn/-/wikis/home, and then navigate to the [downloads](https://gitlab.com/movie_thumbnailer/mtn/-/releases) the direct linux and windows binaries are pointing to _bitbucket_.

I'm not sure what I could do about this, or what info you want me to provide. 😕

Reply:
Both repos are from wahibre

Reply:
Thank you for explaining this ISV's atypical hosting configuration.